src/toolchain/export-android.md

```diff
@@ -0,0 +1,105 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+NOTE: Keep track of where you install the Android Command Line Tools and JDK 17
```

Comment:
Several things:
- Please start a page with a h1 heading (Markdown: `# Title`) and add it to SUMMARY.md. 
- The h1 should be followed by an intro section, which is followed by a TOC (table of contents) comment.
- Note blocks should be inside "admonish" tags with a title.

You can check how other pages do it 🙂

Reply:
The `NOTE` here still needs an admonish block, see previous message :slightly_smiling_face: